### PR TITLE
 js1.3 problem solution

### PR DESCRIPTION
**Pull request by pxa4576**

### CODE REVIEW

File: curriculum/js1/3.js [hunk 1]
```diff
@@ -5,8 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a,result="",i=0) => {
+  if(a===i)
+    return result;
+  else
```
Comment (songz) [inline curriculum/js1/3.js]: since your if logic is returning, you don't need else statement.

File: curriculum/js1/3.js [hunk 2]
```diff
@@ -5,8 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a,result="",i=0) => {
```
Comment (songz) [inline curriculum/js1/3.js]: whats the point of having an i counter? You can just decrement a to 0

Comment (pxa4576) [reply curriculum/js1/3.js]: how do i decrement a. I don't know how to do that? I had i as a counter

Comment (pxa4576) [reply curriculum/js1/3.js]: i had i as a counter

**Review by songz — APPROVED** [review]

**Pull request merged**